src/main/java/mate.academy/DirectoryScanner.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import mate.academy.lib.Component;
+
+public class DirectoryScanner {
+
+    public Map<Class<?>, Class<?>> getComponents(String directoryName, String packageName) {
```

Comment:
it's very cool that you made own directory scanner and other things, but please make your solution pass tests :)